src/goto-programs/goto_inline_class.cpp

```diff
@@ -231,14 +230,6 @@ void goto_inlinet::replace_return(
         assignment->source_location=it->source_location;
         assignment->function=it->function;
 
-        if(constrain.is_not_nil() && !constrain.is_true())
```

Comment:
Just to check, since we're removing code from inside `#if 0`, can the whole `#if 0` block be deleted?